engo_glfw.go

```diff
@@ -317,7 +323,8 @@ func SetVSync(enabled bool) {
 	}
 }
 
-func init() {
+// Initializes application
+func InitGl() {
 	runtime.LockOSThread()
```

Comment:
This method should not be here, but when opening the OpenGL window. And maybe even related to OpenGL calls? Not sure about that. Having it here does not make sense directly, IMHO. 

Also, not PR related, but if this one is moved as well, this method is empty and can be removed entirely. No need to rename or discuss about `init` vs `InitGl`.